packages/viewer/src/lib/stream/stream.ts

```diff
@@ -581,6 +581,15 @@ export class ViewerStream extends StreamApi {
     }
   }
 
+  private getDimensions(): Dimensions.Dimensions {
+    if (Dimensions.area(this.dimensions) === 0) {
```

Comment:
`... <= 0`?

Reply:
Personally, I think I'd rather let FSS throw an exception in the case of `<= 0`, but open to other thoughts there. This `=== 0` case is a valid state in the UI (`display: none` or auto-sized to  a width/height of 0), where I don't think there's a valid case for negative dimensions. Whether that exception is helpful today though is another question, so I could add a case to throw a custom error here